finetune/lora.py

```diff
@@ -109,15 +109,15 @@ def setup(
         seed,
         Config.from_name(
             name=checkpoint_dir.name,
-            r=lora_r,
```

Comment:
The renaming was needed to simplify the logic we have internally to read the config from the saved hyperparameters. Previously, the arguments for the config were named differently than the arguments for the lora.py script.